Symfony/CS/DocBlock/Annotation.php

```diff
@@ -37,7 +37,7 @@ class Annotation
     /**
      * Create a new line instance.
      *
-     * @param Line[] $lines
+     * @param array $lines
```

Comment:
> These docblocks aren't for phpstorm, they're for people.

sure are :)  I was using the `Inspect code` option of PHPStorm and that is why I mentioned it, I didn't mean to imply to change the code to keep PHPStorm happy. I just missed this case.

@stof 8.0.3 on Ubuntu